packages/client/modules/userDashboard/components/OrgBilling/NewBillingLeaderMenu.tsx

```diff
@@ -0,0 +1,118 @@
+import React, {forwardRef, useMemo} from 'react'
+import Menu from '../../../../components/Menu'
+import MenuItem from '../../../../components/MenuItem'
+import MenuItemLabel from '../../../../components/MenuItemLabel'
+import {MenuProps} from '../../../../hooks/useMenu'
+import {useFragment} from 'react-relay'
+import graphql from 'babel-plugin-relay/macro'
+import {
+  NewBillingLeaderMenu_organization$data,
+  NewBillingLeaderMenu_organization$key
+} from '~/__generated__/NewBillingLeaderMenu_organization.graphql'
+import Avatar from '../../../../components/Avatar/Avatar'
+import styled from '@emotion/styled'
+import TypeAheadLabel from '../../../../components/TypeAheadLabel'
+import useFilteredItems from '../../../../hooks/useFilteredItems'
+import {EmptyDropdownMenuItemLabel} from '../../../../components/EmptyDropdownMenuItemLabel'
+import SetOrgUserRoleMutation from '../../../../mutations/SetOrgUserRoleMutation'
+import useAtmosphere from '../../../../hooks/useAtmosphere'
+import useMutationProps from '../../../../hooks/useMutationProps'
+
+const AvatarBlock = styled('div')({
+  paddingRight: 32
+})
+
+interface Props {
+  menuProps: MenuProps
+  organizationRef: NewBillingLeaderMenu_organization$key
+  newLeaderSearchQuery: string
+}
+
+const getValue = (
```

Comment:
-1 because it's far from the usage, we should chose a more meaningful name, or just inline the code.